go/proto/modelzooserver.proto

```diff
@@ -133,4 +137,8 @@ service ModelZooServer {
     rpc DropModel (ReleaseModelRequest) returns (ReleaseResponse);
     // Download a model
     rpc DownloadModel (ReleaseModelRequest) returns (stream DownloadModelResponse);
+
+    // Get model metadata of a trained model.
+    // Only valid when SQLFLOW_USE_EXPERIMENTAL_CODEGEN=true currently.
+    rpc GetModelMeta (ReleaseModelRequest) returns (GetModelMetaResponse);
```

Comment:
Can return string directly?

Reply:
It seems that we cannot return the string directly. Otherwise, `go generate` would cause the error like:
```
modelzooserver.proto:143:53: Expected message type.
go/proto/proto.go:16: running "protoc": exit status 1
```